calcs/vmg_wind_stw.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function (app) {
+  return {
+    group: 'course data',
+    optionKey: 'vmg_Wind_STW',
+    title:
+      'Velocity Made Good to wind (based on wind.angleTrueWater and speedThroughWater)',
+    derivedFrom: [
+      'environment.wind.angleTrueWater',
+      'navigation.speedThroughWater'
+    ],
+    calculator: function (trueWindAngle, speedThroughWater) {
+      var vmg_wind = Math.cos(trueWindAngle) * speedThroughWater
+      if (vmg_wind < 0) {
```

Comment:
Why the if statement? Even if the code you use as base has something foolish let's not propagate it...

Reply:
Good point - I could clean up the base code (vmg_wind.js) if you like.